packages/features/ee/billing/api/webhook/index.ts

```diff
@@ -7,7 +7,7 @@ import { stripeWebhookHandler } from "./__handler";
 const handlers = {
   "payment_intent.succeeded": () => import("./_payment_intent.succeeded"),
   "customer.subscription.deleted": () => import("./_customer.subscription.deleted"),
-  "customer.subscription.updated": () => import("./_customer.subscription.updated"),
+  "customer.subscription.updated": () => import("./_customer.subscription.updated/_handler"),
```

Comment:
We moved the main handler into it's own folder to organizer the handler for the update event and product specific handlers.